plugins/available/vagrant-as-root.plugin.bash

```diff
@@ -0,0 +1,4 @@
+# Simple function for vagrant ssh that immediately switches to root.
+function vsshr() {
```

Comment:
Please use a more descriptive name for this function, like `vagrant-as-root`. Once we have that function, you can add an alias to make it shorter.

Please also add the meta tags for the functions, e.g. like here: https://github.com/Bash-it/bash-it/blob/master/plugins/available/docker-compose.plugin.bash#L5

These tags are used by the various help functions to show what the function is about.